resources/js/components/dashboard/components/IndividualSponsor.tsx

```diff
@@ -145,13 +145,12 @@ class IndividualSponsor extends Component<IIndividualSponsorProps & RouteCompone
             let logoUrl: string | undefined = portalInfo.files.find((x: IAssetInformation) => { return x.name.toLowerCase().includes("logo") });
             if (!logoUrl || logoUrl === undefined) {
                 logoUrl = "https://" + window.location.hostname + "/images/no-logo-sponsor.png";
+            } else {
+                logoUrl = logoUrl.url;
```

Comment:
Yeah, so I had a question about this before, so that's why I commented it expecting something to break, but nothing did.

So you are setting `logoUrl` to `string | undefined`, but then use it as an object `logoUrl.url`. Shouldn't types be different? 😅

Reply:
Wow yeah definitely poor typing there, mucked it up for sure. 

`logoUrl` should be `IAssetInformation`

Reply:
Ah, this makes more sense 😅. I was like - seems okay to call `logoUrl.url`, but the `.url` field of a string 💣